cohort-engine-api-web/pom.xml

```diff
@@ -273,5 +279,49 @@
 			<version>1.16.0</version>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
+			<groupId>com.ibm.websphere.appserver.api</groupId>
+			<artifactId>com.ibm.websphere.appserver.api.jaxrs20</artifactId>
+			<version>1.0.39</version>
+			<scope>provided</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>com.fasterxml.jackson.core</groupId>
+			<artifactId>jackson-databind</artifactId>
+			<version>2.11.2</version>
+			<type>jar</type>
+			<scope>compile</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>ch.qos.logback</groupId>
+			<artifactId>logback-core</artifactId>
+			<version>1.2.3</version>
+			<scope>compile</scope>
+		</dependency>
```

Comment:
We don't want a compile-time dependency on a specific log implementation. This could be "test" scope as needed, but I've been using slf4j-simple as the test binding in other projects.

Reply:
I had to add this for the dependency analyzer, I'll try removing it again and see what pops up.